### PR TITLE
Add/GitHub release assets

### PR DESCRIPTION
**Pull request by Dinek007**

### CODE REVIEW

File: .github/workflows/build-release.yml [hunk 1]
```diff
@@ -159,3 +203,28 @@ jobs:
...
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+
+      - name: Extract version
+        id: extract_version
+        uses: Saionaro/extract-package-version@v1.1.1
+        with:
+          path: packages/desktop
+
+      - name: "Show names"
+        run: cd packages/desktop/dist && dir
```
Comment (vinkabuki) [inline .github/workflows/build-release.yml:220]: Do we need this?

**Review by vinkabuki — APPROVED** [review]

File: .github/workflows/build-release.yml [hunk 2]
```diff
@@ -55,6 +55,28 @@ jobs:
           source_dir: 'packages/desktop/dist/alpha-linux.yml'
           destination_dir: ''
 
+      - name: Get release
+        id: get_release
+        uses: bruceadams/get-release@v1.2.3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+
+      - name: get-npm-version
```
[the remark this answers is not in the document]
Comment (vinkabuki) [reply .github/workflows/build-release.yml:64]: Is that npm version or release version?

**Review by vinkabuki — APPROVED** [review]

File: .github/workflows/build-release.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (Dinek007) [reply .github/workflows/build-release.yml:64]: ye i agree the name of this action is wired but it gives us the release version,
here doc https://github.com/marketplace/actions/get-current-package-version

File: packages/desktop/package.json [hunk 3]
```diff
@@ -68,10 +68,11 @@
   },
   "homepage": "https://github.com/TryQuiet",
   "@comment version": "To build new version for specific platform, just replace platform in version tag to one of following linux, mac, windows",
-  "version": "0.8.3-alpha.0",
+  "version": "0.8.3-alpha.18",
```
Comment (siepra) [inline packages/desktop/package.json]: I'm not sure if we want to bump it this way

Comment (Dinek007) [reply packages/desktop/package.json]: u should refresh the changes, i reverted that change

**Review by siepra — APPROVED** [review]

**Review by EmiM — APPROVED** [review]

**Pull request merged**